src/resolve/resolved-varlink.c

```diff
@@ -100,6 +100,20 @@ static void vl_on_disconnect(VarlinkServer *s, Varlink *link, void *userdata) {
         dns_query_complete(q, DNS_TRANSACTION_ABORTED);
 }
 
+static void vl_on_notification_disconnect(VarlinkServer *s, Varlink *link, void *userdata) {
+        Manager *m = userdata;
```

Comment:
`Manager *m = ASSERT_PTR(userdata);` (and remove the assert(m) below)

Reply:
not addressed

Reply:
Fixed in latest patch